src/data/storage-area.cpp

```diff
@@ -1,11 +1,13 @@
-#include <core.h>
 #include <dicom-file.h>
 #include <plugin-configure.h>
 #include <db-interface.h>
 //#include <job-queue.h>
 
 #include <filesystem>
 #include <fstream>
+#include <iostream>
+#include <job-queue.h>
+#include "dicom/dicom-checksum.h"
```

Comment:
I guess I missed one

Reply:
What's the difference between `< >` and `" "`besides cleanliness?

Reply:
`"..."` is for relative paths and `<...>` are for headers that are in the search path(s) known by the compiler. https://stackoverflow.com/a/21594/1327399

Sometimes you'll see people do stuff like `#include "../../../library_A/gumbo.h"`. In the old days that was probably the norm for codebase headers as that's what that SO answer implies (having been written in 2008)